pyproject.toml

```
@@ -207,6 +260,48 @@ known-first-party = [
 ]
 relative-imports-order = "closest-to-furthest"
 
+[tool.ruff.lint.flake8-quotes]
+docstring-quotes = "double"
+multiline-quotes = "single"
+inline-quotes = "single"
+avoid-escape = false
+
+[tool.ruff.lint.pep8-naming]
+classmethod-decorators = ["yt_dlp.utils.classproperty"]
```

and if we want to go with the convention of not using single-line arrays for any array that may have more than 1 element in the future:
```suggestion
classmethod-decorators = [
    "yt_dlp.utils.classproperty",
]
```